webapp/sources/rudder/rudder-web/src/main/javascript/rudder/rudder-datatable.js

```diff
@@ -1473,14 +1478,15 @@ function createNodeTable(gridId, refresh) {
     , "dom": ' <"dataTables_wrapper_top newFilter "<"#first_line_header" f <"dataTables_refresh"> <"#edit-columns">> <"#select-columns"> >rt<"dataTables_wrapper_bottom"lip>'
   };
 
-  $.ajax({
-    type: 'GET',
-    url: "/rudder/secure/api/scores/list",
-    success: function (data) {
-      scoreList = data.data
-    },
-  });
-
+  if (scoreList.length === 0) {
+    $.ajax({
+      type: 'GET',
+      url: "/rudder/secure/api/scores/list",
+      success: function (data) {
+        scoreList = data.data
+      },
+    }).done( function() {refresh(gridId)} )
```

Comment:
This does a 2nd API request for node details every time we create the table, is there a way without the `refresh` ?